src/components/Home.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable max-len */
+import React, { useState } from 'react';
```

Comment:
- [ ] Kindly do not disable linter because it enforces code quality. Instead, you should fix the issues raised.

Reply:
This change has been implemented well